Engine/source/scene/pathManager.cpp

```diff
@@ -299,15 +303,33 @@ void PathManager::getPathPosition(const U32 id,
 
    // Ok, query holds our path information...
    F64 ms = msPosition;
-   if (ms > mPaths[id]->totalTime)
-      ms = mPaths[id]->totalTime;
+
+   //Looping vs. non-looping splines
```

Comment:
Nit-picking here but I prefer having a space between // and comment text:

```suggestion
   // Looping vs. non-looping splines
```